integration_tests/mockApis/csipApi.ts

```diff
@@ -764,6 +764,10 @@ const stubPostIdentifiedNeedFail = () => {
   })
 }
 
+const stubPatchReviewSuccess = () => {
+  return createBasicHttpStub('PATCH', '/csip-api/csip-records/plan/reviews/[a-zA-Z0-9-]+-[a-zA-Z0-9-]+', 200)
+}
```

Comment:
`[a-zA-Z0-9-]+-[a-zA-Z0-9-]+` do we need to repeat it twice?

Reply:
so that it doesn't capture the `/attendees` endpoint as well